### ISSUE TITLE
"Change all occurrences" is missing from context menus

### ISSUE THREAD

**Issue by iansan5653** [microsoft/vscode#85255]
Version: 1.41.0-insider (system setup)
Commit: bf7d03bf88732af0b46e473c663c49e18e839f9c
Date: 2019-11-15T05:40:12.201Z
Electron: 6.1.4
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363

There used to be a "Change all occurrences" option in the context menu that would activate cursors at every instance of the selection in the current file. Now it seems to be missing as of recently.

**Comment by iansan5653** [comment]
I see there's still a `Selection` -> `Select All Occurrences` option, but is there any way to get it back in the context menu?

**Comment by gjsjohnmurray** [comment]
Apparently done by @jrieken in https://github.com/microsoft/vscode/commit/5c27cdccf8621e212d0113b18f4ef07714c28d3f

**Comment by jrieken** [comment]
We did that on purpose and only when there is a rename provider - which we think is the better alternative. @iansan5653 What the use case for "change all" over "rename"?

**Comment by iansan5653** [comment]
For renaming symbols alone,  "change all" sometimes is useful when the language provider is hanging (happens often with Python) and I know the symbol is only used in the current file.

But it's also useful for tons of other cases:

For example, in this snippet I might want to add `: void` to all the functions where I didn't define a return type. I can highlight one instance of `() {` and then use "Change all occurrences" and then change them to `(): void {`.
```ts
function example0(): number {
  // ...
}

function example1(): string {
  // ...
}

function example2() {
  // ...
}

function example3(): string {
  // ...
}

function example4() {
  // ...
}
```

The above could also be done with find and replace. However, having multiple cursors in use is compatible with things like the "Increment Selection" extension, which is exactly how I generated the above example - I wrote one `example()` function, copied it a couple times, selected `example()`, change all, used the arrow keys to put the cursor before the parentheses and then put the numbers with Increment Selection.

Here's another example. Say I'm fixing some old code. I want to change `var` to `const`, add semicolons, and change `example` to `sample`. I can select the first `var example`, change all occurrences, type `const sample`, press <kbd>End</kbd>, type `;` and I'm done.
```js
var exampleString = "sample"
// This stops me from just using column select
var exampleBoolean = true
var exampleNumber = 45
```
